website/src/app/kb/deploy/gateways/readme.mdx

```diff
@@ -121,6 +117,11 @@ balancing.
   portal for audit trail and logging purposes.
 </Alert>
 
+### Deploy using Terraform
+
+For examples on how to deploy multiple Gateways using Terraform, see our example
+[Terraform modules](https://github.com/firezone/firezone/tree/main/terraform/examples).
```

Comment:
They are not *modules* per se. Even though everything is a module - they can't be just included without changes, let's just call them examples and call modules all the heavily reusable things.
```suggestion
For examples on how to deploy multiple Gateways using Terraform, see our 
[examples](https://github.com/firezone/firezone/tree/main/terraform/examples).
```